### PR TITLE
(FM-8232) Convert to litmus

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,23 +24,117 @@ matrix:
...
+      sudo: required
+      stage: acceptance
+    -
+      bundler_args:
+      dist: trusty
+      env: PLATFORMS=el6_puppet6
+      rvm: 2.5.3
+      before_script:
+      - bundle exec rake 'litmus:provision_list[travis_el6]'
+      - bundle exec rake 'litmus:install_agent'
```
Comment (florindragos) [inline .travis.yml]: `[puppet6]` ?

File: .travis.yml [hunk 2]
```diff
@@ -24,23 +24,117 @@ matrix:
...
+      sudo: required
       stage: acceptance
+    -
+      bundler_args:
+      dist: trusty
+      env: PLATFORMS=el7_puppet6
+      rvm: 2.5.3
+      before_script:
+      - bundle exec rake 'litmus:provision_list[travis_el7]'
+      - bundle exec rake 'litmus:install_agent'
```
Comment (florindragos) [inline .travis.yml]: `[puppet6]`

File: .travis.yml (hunk 1 above)
Comment (tphoney) [reply .travis.yml]: not needed, by default it will do puppet6. i dont think i can take another travis run

File: .travis.yml (hunk 2 above)
Comment (tphoney) [reply .travis.yml]: not needed, by default it will do puppet6. i dont think i can take another travis run

**Pull request merged**